src/Plugins/ComplexCore/src/ComplexCore/Filters/Algorithms/ImportVolumeGraphicsFile.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include "ComplexCore/ComplexCore_export.hpp"
+
+#include "complex/DataStructure/DataPath.hpp"
+#include "complex/DataStructure/DataStructure.hpp"
+#include "complex/Filter/IFilter.hpp"
+#include "complex/Parameters/ArrayCreationParameter.hpp"
+#include "complex/Parameters/FileSystemPathParameter.hpp"
+#include "complex/Parameters/StringParameter.hpp"
+
+namespace complex
+{
+
+struct COMPLEXCORE_EXPORT ImportVolumeGraphicsFileInputValues
+{
+  FileSystemPathParameter::ValueType VGHeaderFile;
+  DataPath ImageGeometryPath;
+  std::string CellAttributeMatrixName;
+  std::string DensityArrayName;
+  std::filesystem::path VGDataFile;
```

Comment:
You should inlcude the <filesystem> header if you are using std::filesystem in the header. Along with <string> and strip out includes that are not directly being used in this header like ArrayCreationParameter.hpp